proxy/Dockerfile

```diff
@@ -17,7 +17,8 @@ FROM gcr.io/inverting-proxy/agent@sha256:d0a06a247bb443f9528356a1341cadfa4c4479a
 
```

Comment:
Do you know if the inverting-proxy image has all the licenses?

Reply:
yes it does have the licenses.